sdkit/utils/__init__.py

```diff
@@ -48,6 +43,45 @@
     take_memory_snapshot,
 )
 
-from .http_utils import (
+from sdkit.utils.http_utils import (
     download_file,
-)
\ No newline at end of file
+)
+
+
+log = logging.getLogger('sdkit')
+LOG_FORMAT = '%(asctime)s.%(msecs)03d %(levelname)s %(threadName)s %(message)s'
+logging.basicConfig(
+        level=logging.INFO,
+        format=LOG_FORMAT,
+        datefmt="%X"
+)
+
+__all__ = [
```

Comment:
Why is this necessary? The purpose the separate init file in this module is to only import the things that we want to expose. Why do we need a redundant `__all__`?